src/Mono.Android/Mono.Android.targets

```diff
@@ -65,13 +65,12 @@
     </ItemGroup>
     <PropertyGroup>
       <ApiMerge>..\..\bin\Build$(Configuration)\api-merge.exe</ApiMerge>
-      <_Profiles>@(_AndroidProfile->'%(Identity)', ' ')</_Profiles>
-      <_Glob>-s '..\..\bin\Build$(Configuration)\api\api-*.xml.in'</_Glob>
-      <_LastProfile>--last-description=..\..\bin\Build$(Configuration)\api\api-$(AndroidPlatformId).xml.in</_LastProfile>
-      <_Out>-o "$(IntermediateOutputPath)mcw\api.xml"</_Out>
+      <_ConfigurationFile>..\..\bin\Build$(Configuration)\merge-configuration.xml</_ConfigurationFile>
+      <_ConfigurationInputBaseDirectory>..\..\bin\Build$(Configuration)\api\</_ConfigurationInputBaseDirectory>
+      <_ConfigurationOutputBaseDirectory>obj\$(Configuration)\</_ConfigurationOutputBaseDirectory>
```

Comment:
I think this should be `<_ConfigurationOutputBaseDirectory>$(IntermediateOutputPath)</_ConfigurationOutputBaseDirectory>`

The build now fails with:
```
_GenerateApiDescription:
  Creating directory "obj/Release/monoandroid10/android-29/mcw".
  mono --debug=casts ../../bin/BuildRelease/api-merge.exe -config=../../bin/BuildRelease/merge-configuration.xml -config-input-dir=../../bin/BuildRelease/api/ -config-output-dir=obj/Release/
  api-merge: writing output obj/Release/android-19\mcw\api.xml...
  api-merge: writing output obj/Release/android-20\mcw\api.xml...
  api-merge: writing output obj/Release/android-21\mcw\api.xml...
  api-merge: writing output obj/Release/android-22\mcw\api.xml...
  api-merge: writing output obj/Release/android-23\mcw\api.xml...
  api-merge: writing output obj/Release/android-24\mcw\api.xml...
  api-merge: writing output obj/Release/android-25\mcw\api.xml...
  api-merge: writing output obj/Release/android-26\mcw\api.xml...
  api-merge: writing output obj/Release/android-27\mcw\api.xml...
  api-merge: writing output obj/Release/android-28\mcw\api.xml...
  api-merge: writing output obj/Release/android-29\mcw\api.xml...
  api-merge: writing output obj/Release/android-R\mcw\api.xml...
_GenerateBinding:
  mono --debug=casts "/Users/builder/azdo/_work/3/s/xamarin-android/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/generator.exe" --public --product-version=7 --api-level=29 -o "obj/Release/monoandroid10/android-29/mcw" --codegen-target=XAJavaInterop1 --fixup=metadata --preserve-enums --enumflags=enumflags --enumfields=map.csv --enummethods=methodmap.csv --enummetadata=obj/Release/monoandroid10/android-29/mcw/enummetadata --apiversions="/Users/builder/Library/Android/sdk/platforms/android-29/data/api-versions.xml" --annotations="/Users/builder/Library/Android/sdk/platform-tools/api/annotations.zip" --assembly="Mono.Android, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null" --type-map-report=obj/Release/monoandroid10/android-29/mcw/type-mapping.txt  --enumdir=obj/Release/monoandroid10/android-29/mcw obj/Release/monoandroid10/android-29/mcw/api.xml
  
  Unhandled Exception:
  System.IO.FileNotFoundException: Could not find file "/Users/builder/azdo/_work/3/s/xamarin-android/src/Mono.Android/obj/Release/monoandroid10/android-29/mcw/api.xml"
  File name: '/Users/builder/azdo/_work/3/s/xamarin-android/src/Mono.Android/obj/Release/monoandroid10/android-29/mcw/api.xml'
```

Reply:
I got closer to using `$(IntermediateOutputPath)`.  The issue is that the value is for the first version it builds: `obj/Release/monoandroid10/android-29/`, and we are building for all versions, so we need to strip off the `android-29` and then generate the correct paths (`/android-28`, `/android-27`, etc.).